### ISSUE TITLE
uestions for HY/SKME

### ISSUE THREAD

**Issue by diazrenata** [diazrenata/scadsanalysis#51]
### General

**Submission details**
- [x] Formatting: font? page numbers?
- [x] Corresponding author marked correctly? 
- [x] Address for correspondence?
- [x] COI statement on front page?
- [x] Conflicted/suggested reviewers? I have a blank here. Both in terms of, do we provide this list or is it obvious to the editor, and if it's important that we provide it, who's on each list? Would assume Ethan, Locey, Kitzes, Newman can't review, but all are in acknowledgements/heavily cited in the text.

**Data**
- [x] I want to be sure we attribute things properly in the data accessibility statement, text, and refs. The data for the analysis are (mostly) the same as Baldridge (2016), the exception being that I accessed Misc Abund from figshare. That version **may** match the data used in Baldridge (2016); I re-accessed it because it's a more heterogeneous dataset than the others and I wanted to be able to see clearly what I was getting (instead of the processed versions). For the others, I downloaded the .csvs from weecology/sadcomparison. The data on GitHub are identical to what is on Zenodo. Details at #52 
- [x] Is it OK that I'm getting the data from GitHub and figshare, rather than Zenodo? ([GH data exactly matches Zenodo](https://github.com/diazrenata/scadsanalysis/blob/master/analysis/reports/accessory/compare_zenodo_data.md), for what it's worth)
- [x] Is it OK if we then re-archive on Zenodo?

### Hao

- [x] Have I got the right affiliation? What's your preferred email?
- [x] Is there anything you'd like to put in the acknowledgements?
- [x] Check description of sampling algorithm now in Methods
- [x] Should algorithm vignette be a supplement?

More to sign off on/close the loop -
- [x] Minor language changes in comments (flagged)
- [x] Updates to figures & figure legends

### Morgan

- [x] Signing off on language changes in comments (flagged)

Some figure changes:
- [x] Removed fig. 1 (distribution of datasets in sXn space) and put it in the supplement
- [x] Changes to fig. 4 to add breadth index - too dense now?

**Comment by ha0ye** [comment]
> ### General
> **Submission details**
> 
> * [ ]  Formatting: font? page numbers?

Looks fine to me. You may want to include the manuscript title in the header, so that if reviewers print out and shuffle papers, they can reassemble.

> * [ ]  Corresponding author marked correctly?

yes

> * [ ]  Address for correspondence?

110 Newins-Ziegler Hall
PO Box 110430,
Gainesville, FL 32611-0430
(should direct to the WEC mailboxes)

> * [ ]  COI statement on front page?

doesn't hurt to include it

> * [ ]  Conflicted/suggested reviewers? I have a blank here. Both in terms of, do we provide this list or is it obvious to the editor, and if it's important that we provide it, who's on each list? Would assume Ethan, Locey, Kitzes, Newman can't review, but all are in acknowledgements/heavily cited in the text.

Usually conflicted means "subject area nemesis", so likely not those folks. You could do a quick a lit search on species abundance distribution and cross with theoretical ecology to identify who is working in the area recently. "Big" names are fine, they often know of colleagues to recommend.

> 
> **Data**
> 
> * [ ]  Is it OK that I'm getting the data from GitHub and figshare, rather than Zenodo? ([GH data exactly matches Zenodo](https://github.com/diazrenata/scadsanalysis/blob/master/analysis/reports/accessory/compare_zenodo_data.md), for what it's worth)
> * [ ]  Is it OK if we then re-archive on Zenodo?

figshare is archival and fine, GitHub is a little more iffy - you may not be able to redeposit on Zenodo
I guess make sure you have a personal copy of anything on GitHub - maybe @ethanwhite has ideas?

> ### Hao
> * [ ]  Have I got the right affiliation? What's your preferred email?

yes, haoye@ufl.edu

> * [ ]  Is there anything you'd like to put in the acknowledgements?

do add something like "HY's time was supported by Gordon and Betty Moore Foundation’s Data-Driven Discovery Initiative, Grant GBMF4563, awarded to E. P. White"

> * [ ]  Should algorithm vignette be a supplement?

Makes sense to me to include those details.

**Comment by ethanwhite** [comment]
Sounds like all of the data is archived already, it's just that you're accessing it from non-archival locations in the code? If so I think that's fine just make sure to cite the archival copies in the paper and consider providing links in comments in the code. The reason to provide links in the comments is that if the non-archival version disappears at some point then this will make it easier for folks to figure out how to update to access the data elsewhere.

Rearchiving is fine since everything is openly licensed. There just needs to be attribution to the original source. Sometimes we'll add documentation that basically says - "this is a copy of the data for reproducibility, but if you plan on using it for new analyses you should go to the primary source". This is both a courtesy since the original provider may want to track usage and useful in that you'd rarely update a dataset that you've provided for reproducing a paper, but the original dataset producer maybe be updating with error fixes, new data, etc.

**Comment by diazrenata** [comment]
Thanks @ethanwhite, that’s right! Everything is on Zenodo/figshare, but the repo is set up to pull it from Github instead. I don’t feel great retooling the repo to get it from Zenodo at this point, but have checked that the Zenodo versions match and can include links to those in the code.

**Comment by ha0ye** [comment]
> * [ ]  Check description of sampling algorithm now in Methods

Updated to better match terminology with the text:
https://github.com/ha0ye/feasiblesads/tree/algo-vignette

PDF form:
https://github.com/ha0ye/feasiblesads/blob/algo-vignette/vignettes/sampling_algorithm.pdf

**Comment by ha0ye** [comment]
https://wec.ifas.ufl.edu/about-wec/contact-information/ has phone and fax numbers.

**Comment by diazrenata** [comment]
A few more as I work through EL's portal...

- [x] EL offers the option of applying for Open Data or Open Materials [badges](https://osf.io/tvyxz/wiki/1.%20View%20the%20Badges/). I think this project is 100% for both of those, unless there are objections I'm not aware of? We'd need to archive it and provide a doi at the time of submission. I still have things I'd like to tweak (wanted to have a fully reproducible example with a subset that does not reuire a supercomputer, and wanted to make a step-by-step guide for how to run the whole analysis). I'd planned to do this while the ms was in review, but could either step it up and do it now, but I wonder if we can actually archive a version now and add a new version/release with the final polishes later? 
- [x] EL is doing "Transparent Peer Review" with an opt-out option. I suppose I don't mind going with the transparent option, unless there's a reason to opt out?

> ***TRANSPARENT PEER REVIEW***

> Ecology Letters is participating in a pilot on Peer Review Transparency. By submitting to this journal, authors agree that the reviewer reports, their responses, and the editor's decision letter will be linked from the published article to where they appear on Publons in the case that the article is accepted. Authors have the opportunity to opt out during submission, and reviewers may remain anonymous unless they would like to sign their report.

- [x] Do any of us have restrictions on what kind of licensing we can agree to? See "The submitting author is expected to consult all authors to find out whether any of their funders has a policy that restricts which kinds of license they can sign, for example if the funder is a member of Coalition S.".  I'd never heard of this :eyes:
- [x] EL offers the option to put a preprint on Authorea. I'm generally pro-preprint, unless there are factors I'm not catching?

**Comment by ha0ye** [comment]
> A few more as I work through EL's portal...
> 
> * [ ]  EL offers the option of applying for Open Data or Open Materials [badges](https://osf.io/tvyxz/wiki/1.%20View%20the%20Badges/). I think this project is 100% for both of those, unless there are objections I'm not aware of? We'd need to archive it and provide a doi at the time of submission. I still have things I'd like to tweak (wanted to have a fully reproducible example with a subset that does not reuire a supercomputer, and wanted to make a step-by-step guide for how to run the whole analysis). I'd planned to do this while the ms was in review, but could either step it up and do it now, but I wonder if we can actually archive a version now and add a new version/release with the final polishes later?

:+1: Zenodo DOIs are versioned, so you can release now and update later - https://guides.github.com/activities/citable-code/

(all my objections to badges relate to performative social signaling)

> * [ ]  EL is doing "Transparent Peer Review" with an opt-out option. I suppose I don't mind going with the transparent option, unless there's a reason to opt out?
> 
> > _**TRANSPARENT PEER REVIEW**_
> 
> > Ecology Letters is participating in a pilot on Peer Review Transparency. By submitting to this journal, authors agree that the reviewer reports, their responses, and the editor's decision letter will be linked from the published article to where they appear on Publons in the case that the article is accepted. Authors have the opportunity to opt out during submission, and reviewers may remain anonymous unless they would like to sign their report.

Potentially there are some reviewers who may be reluctant to have their reviews published, but I don't see that as a major problem here.

> * [ ]  Do any of us have restrictions on what kind of licensing we can agree to? See "The submitting author is expected to consult all authors to find out whether any of their funders has a policy that restricts which kinds of license they can sign, for example if the funder is a member of Coalition S.".  I'd never heard of this 👀

By my reading:
* Moore requires open access deposition of the post-print article within 12 months - https://www.moore.org/docs/default-source/Grantee-Resources/open-access-policy-2017.pdf?sfvrsn=2
* Wiley copyright transfer allows deposition of the accepted version (NOT EL-formatted pdf) after 12 months - https://authorservices.wiley.com/asset/Copyright-Transfer-Agreement-Sample.pdf
* UF's insitutional repository or bioRxiv/OSF would work as a repository.
* UF's author rights (formerly known as "open access") policy is not yet in effect, but is compatible with the above - https://digitalpartnerships.uflib.ufl.edu/uf-author-rights-policy/

> * [ ]  EL offers the option to put a preprint on Authorea. I'm generally pro-preprint, unless there are factors I'm not catching?

No specific objections - prefer something not controlled by Wiley because vertical integration of publishing infrastructure, yada yada...

**Comment by ethanwhite** [comment]
In general I definitely recommend posting preprints to bioRxiv. It is the standard location in ecology, people pay attention to it, they understand what that location implies, and, as Hao points out, it's not a Wiley owned destination (and Wiley doesn't have a great record of keeping things going in the long-run in my experience). So unless you have a specific reason to do something else I'd post it there separately at the time of submission.

**Comment by skmorgane** [comment]
I've put Ethan! or Hao! if I agreed with their responses and had nothing further to add.
> ### General
> 
> **Submission details**
> 
>     * [ ]  Formatting: font? page numbers?
> 
Yes, add page numbers (for similar reasons to Hao's add header). Few journals have font constraints (would be in author giudelines if they did) though funding agencies often do for proposals. Don't see any problems with your font. If you're nervous about it for some reason Arial and Times New Roman are pretty standard in ecology so you can use one of those if you'd like.

>     * [x]  Corresponding author marked correctly?
>
Yes
>     * [ ]  Address for correspondence?
> 
Hao!
>     * [x]  COI statement on front page?
> 
Hao!
>     * [ ]  Conflicted/suggested reviewers? I have a blank here. Both in terms of, do we provide this list or is it obvious to the editor, and if it's important that we provide it, who's on each list? Would assume Ethan, Locey, Kitzes, Newman can't review, but all are in acknowledgements/heavily cited in the text.
> 
Conflicted reviewers: Hao! Suggested reviewers: Hao! Also, Erica Newman might be a good suggestion assuming you and she aren't actively working on something together. Sean Connelly might be as well - he was really interested in this space once upon a time.  Jim once told me to see if there's anyone I cite a lot (who we don't have conflicts with) because that's a good sign they should probably review the paper.

> **Data**
> 
>     * [ ]  I want to be sure we attribute things properly in the data accessibility statement, text, and refs. The data for the analysis are (mostly) the same as Baldridge (2016), the exception being that I accessed Misc Abund from figshare. That version **may** match the data used in Baldridge (2016); I re-accessed it because it's a more heterogeneous dataset than the others and I wanted to be able to see clearly what I was getting (instead of the processed versions). For the others, I downloaded the .csvs from weecology/sadcomparison. The data on GitHub are identical to what is on Zenodo. Details at #52
> 
>     * [ ]  Is it OK that I'm getting the data from GitHub and figshare, rather than Zenodo? ([GH data exactly matches Zenodo](https://github.com/diazrenata/scadsanalysis/blob/master/analysis/reports/accessory/compare_zenodo_data.md), for what it's worth)
> 
>     * [ ]  Is it OK if we then re-archive on Zenodo?
> 

Seems like Hao! and Ethan! both covered this.

> 
> ### Morgan
> 
>     * [ ]  Signing off on language changes in comments (flagged)
> 
I either left a comments saying some variant of a thumbs up or making a suggestion (or edits). They are just in places where there was either an active comment or a past comment. And everything I did or suggested was super minor!

> Some figure changes:
> 
>     * [ ]  Removed fig. 1 (distribution of datasets in sXn space) and put it in the supplement

I'm good with this! 

>     * [ ]  Changes to fig. 4 to add breadth index - too dense now?

If I checked the right thing on GitHub it looks good to me!

>A few more as I work through EL's portal...
>
>EL offers the option of applying for Open Data or Open Materials badges. I think this project is 100% for both of those, unless >there are objections I'm not aware of? We'd need to archive it and provide a doi at the time of submission....
Hao!

Looks like the rest (transparent review, preprints, etc) are all Hao! Ethan! 

So I think I've worked through everything?

**Comment by diazrenata** [comment]
Thanks everyone!! I think most things are settled at this point - just a few straggling ends... -

- [x] By my reading, Moore wants the accepted version in a repository **within** 12 months, and EL allows archiving **after** 12 months. Do these count as compatible (if, say, one were to archive it at midnight on the anniversary of its publication 🤔)? Alternatively there's the option to pay...$4800 for OnlineOpen. 👀 
- [x] Totally happy to preprint on biorxiv, unless the journal would object? I don't see anything disallowing this on EL's author guidelines or [Wiley's author services page](https://authorservices.wiley.com/author-resources/Journal-Authors/open-access/preprints-policy.html), but since this is the first time I've tried to figure something like this out, I'm not 100% confident that it's ok?
- [x] Can we suggest Erica Newman as a reviewer if we also thank her in acknowledgements? Our conversations weren't laser-focused on SADs, but she was really helpful about the whole statistical-constraint space and we've corresponded a bit about SADs since we met. Other folks who come to mind because they publish in the same idea space are McGill, O'Dwyer, maybe Chase - but I don't know if any of them count as conflicted or if there's additional nuance I'm not catching.

**Comment by ha0ye** [comment]
I have a few straggling edits to complete and then send back, but they're all minor.

Yes to all of the above.

[EL does take preprints](https://jabberwocky.weecology.org/2014/09/08/ecology-letters-now-allows-preprints-and-why-this-is-a-big-deal-for-ecology/), but you may need to correct an editor if they don't realize that the automated plagiarism detector is finding your own preprint.

**Comment by ethanwhite** [comment]
The easy solution to archival timing is to just make sure to update the preprint at every resubmission. That way, at the point the paper gets accepted the final version is already online.

**Comment by diazrenata** [comment]
Thanks everyone! Apologies for having so many uestions about things that probably seem minor or overly detailed. I'm trying to step carefully because I have the sense there's a set of normal practices that I'm not fully attuned to. I imagine these things seem fairly obvious and unimportant from a more experienced perspective, and I appreciate you all being patient with me!

**Comment by ethanwhite** [comment]
Nothing to apologize for. Learning this sort of stuff is important and there's no good central source for it. That's why we are continually trying to build up the lab wiki (and a lab manual that @skmorgane & I are working on this semester) to help. @skmorgane & I used this discussion to update the [Manuscript Submission Process](https://github.com/weecology/lab-wiki/wiki/Manuscript-Submission-Process) page on the wiki this morning. When you get a chance give it a read through and see if we successfully covered everything and either add or open issues for anything that's still missing that you came across as part of this process. We've also tried to make it more explicit about what we recommend instead of just providing information.
